services/filemanager/miniomanager.go

```diff
@@ -147,7 +160,7 @@ func (manager *MinioManager) GetConfiguredPrefix() string {
 }
 
 func GetMinioConfig(config map[string]interface{}) *MinioConfig {
-	var bucketName, prefix, endPoint, accessKeyID, secretAccessKey string
+	var bucketName, prefix, endPoint, accessKeyID, secretAccessKey, continuationToken string
```

Comment:
I had to make continuation token of this as part of the config because S3 was desinged that way (https://github.com/rudderlabs/rudder-server/blob/master/services/filemanager/s3manager.go#L199) and all the references for ListFilesWithPrefix are written with this implementation in mind. If I start changing it in all the places, it might go out of the scope of this PR right?